WordPressUI/Ghosts/UIView+Ghost.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import UIKit
+
+
+// MARK: - Ghost Animations API
+//
+extension UIView {
```

Comment:
Ghost ALL the views! 👻 👻 👻 👻 👻

Reply:
I kind of had lots of fun picking up a name for the private extension (`+InnerGhost`) LOL